controllers/cloudstackmachine_controller.go

```diff
@@ -55,8 +56,13 @@ type CloudStackMachineReconciler struct {
 	CS     cloud.Client
 }
 
-const requeueTimeout = 5 * time.Second
-const destoryVMRequeueInterval = 10 * time.Second
+const (
+	requeueTimeout           = 5 * time.Second
+	destoryVMRequeueInterval = 10 * time.Second
+)
+
+var AffinityNameAnnotationKey = strings.Join([]string{
```

Comment:
Would like this to be a constant, but Go...

Thought about getting creative with types, but it just doesn't make sense to put that much effort into constantizing this.